foods.html

```diff
@@ -11,8 +11,8 @@
   <ul id='list'></ul>
 
   <form id="food_form">
-    <input type="text" id="name" /> <br />
-    <input type="text" id="calories" />
+    <input type="text" id="name" name="name" required/> <br />
+    <input type="text" id="calories" name="calories" required/>
     <input id="add-food" type="submit" value="Add Food" />
```

Comment:
@kat3kasper are form validations okay, do you think? All story functionality works.